lib/services/metadata.test.js

```diff
@@ -155,4 +155,68 @@ describe(_.startCase(filename), function () {
         });
     });
   });
+
+  describe('checkProps', function () {
+    const fn = lib[this.title];
+
+    it('returns false for a string prop with value false', function () {
+      db.getMeta.returns(Promise.resolve({ published: false }));
+
+      return fn('domain.com/_pages/id/meta', 'published')
+        .then((resp) => {
+          expect(resp).to.equal(false);
+        });
+    });
+
+    it ('returns true for a string prop with value true', function () {
```

Comment:
Oops
```suggestion
    it('returns true for a string prop with value true', function () {
```
Do this for lines 180, 195 and 210